source/index.html.md

```diff
@@ -523,8 +570,41 @@ payment method must be omitted.
     <i>Optional</i> <br />
     A reference to an external object, such as an order number.
   </dd>
+  <dt>credential_on_file[type]</dt>
+  <dd>
+    (installment|recurring|unscheduled|cardholder) <br />
```

Comment:
I'd maybe prefer _cardholder_initiated_.

Reply:
All first in series transactions are "cardholder initiated".